pkg/netdevice/pciNetDevice.go

```diff
@@ -48,7 +50,14 @@ func NewPciNetDevice(dev *ghw.PCIDevice, rFactory types.ResourceFactory, rc *typ
 	nf, ok := rc.SelectorObj.(*types.NetDeviceSelectors)
 	if ok {
 		// Add InfoProviders based on Selector data
-		if nf.IsRdma {
+		if nf.VdpaType != "" {
+			vdpaDev = rFactory.GetVdpaDevice(dev.Address)
+			if vdpaDev == nil {
+				glog.Warningf("No vDPA device found for device %s", dev.Address)
+			} else {
+				infoProviders = append(infoProviders, NewVdpaInfoProvider(nf.VdpaType, vdpaDev))
+			}
+		} else if nf.IsRdma {
```

Comment:
It might be stupid question, as I know virtually nothing about vDPA to be honest - cannot such device use RDMA?

Reply:
No, rdma and vdpa are mutually exclusive ways of consuming a device

Reply:
shall we add a check in initServers to validate resource config does not set both IsRdma and VdpaType ?

also maybe add info in README in the selectors table WDYT?